drivers/charger/charger_bq25180.c

```diff
@@ -140,6 +170,41 @@ static int bq25180_get_charge_current(const struct device *dev,
 	return 0;
 }
 
+static int bq25180_set_charge_voltage(const struct device *dev, uint32_t const_charge_voltage_ua)
```

Comment:
should be `const_charge_voltage_uv`

Reply:
yeah, sorry about that. Forgot it when copy pasting... Fixed now.